docs/usage.md

```diff
@@ -844,6 +844,78 @@ nextflow run download_cache.nf --cadd_cache </path/to/CADD/cache> --cadd_version
 Resource requests are difficult to generalize and are often dependent on input data size. Currently, the number of cpus and memory requested by default were adapted from tests on 5 ICGC paired whole-genome sequencing samples with approximately 40X and 80X depth.
 For targeted data analysis, this is overshooting by a lot. In this case resources for each process can be limited by either setting `--max_memory` and `-max_cpus` or tailoring the request by process name as described [here](#resource-requests). If you are using sarek for a certain data type regulary, and would like to make these requests available to others on your system, an institution-specific, pipeline-specific config file can be added [here](https://github.com/nf-core/configs/tree/master/conf/pipeline/sarek).
 
+## Spark related issues
+
+If you have problems running processes that make use of Spark such as `MarkDuplicates`.
+You are probably experiencing issues with the limit of open files in your system.
+You can check your current limit by typing the following:
```

Comment:
```suggestion
```suggestion
If you have problems running processes that make use of Spark such, for instance, as `MarkDuplicates`, then that might be due to a limit on the number of simultaneously open files on your system.
```

Reply:
tiny issue with your suggestion,